hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/HoodieFileGroupReaderBasedFileFormat.scala

```diff
@@ -211,9 +210,8 @@ class HoodieFileGroupReaderBasedFileFormat(tablePath: String,
     val exclusionFields = new java.util.HashSet[String]()
     exclusionFields.add("op")
     partitionSchema.fields.foreach(f => exclusionFields.add(f.name))
-    val requestedSchema = StructType(requiredSchema.fields ++ partitionSchema.fields.filter(f => mandatoryFields.contains(f.name)))
```

Comment:
This struct type schema needs to be kept and renamed.

Reply:
Changes made:

`dataSchema` (in method signature) -> `dataStructType`
`requestedSchema` -> `requestedStructType`

These changes make sense to me as the type is now described in the variable name and is more descriptive.